src/lib/selections.ts

```diff
@@ -596,6 +611,11 @@ export function canLoftSelection(selection: Selections) {
   )
 }
 
+/**
+ * @deprecated was made for shell guard on modeling machine.
+ * Now the selection command bar argument input handles this.
+ * Leaving until we're sure it's not needed elsewhere.
+ */
```

Comment:
I added Loft and Shell 2w ago and created these. I say we delete!

Reply:
Oh of course, I need to delete all of them I searched for the wrong thing

Reply:
Okay done